packages/gatsby/src/bootstrap/create-graphql-runner.ts

```diff
@@ -43,12 +45,17 @@ export const createGraphQLRunner = (
 
   eventTypes.forEach(type => {
     emitter.on(type, () => {
-      runner = new GraphQLRunner(store)
+      runner = undefined
```

Comment:
I hope this doesn't lead to race conditions where the runner is undefined between firing certain events but where other parts expect it to exist anyways.

Reply:
I don't immediately see how this can happen because we create the runner right before we use it and we don't use it anywhere else on my first glance. What am I missing here? 😄

Reply:
This should be fine, because it's always created on demand if it's not there and only used at that place.